bindings/python/crocoddyl/multibody/contacts/contact-1d.cpp

```diff
@@ -0,0 +1,122 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020-2021, LAAS-CNRS, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/multibody/multibody.hpp"
+#include "crocoddyl/multibody/contacts/contact-1d.hpp"
+#include "python/crocoddyl/utils/deprecate.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeContact1D() {
+  bp::register_ptr_to_python<boost::shared_ptr<ContactModel1D> >();
+
+#pragma GCC diagnostic push  // TODO: Remove once the deprecated FrameXX has been removed in a future release
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Removed these lines.
We used to disable the print of deprecated code.
However, this shouldn't be the case for a new class.